rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "stable"
+channel = "nightly"
```

Comment:
Is there a reason for switching to nightly. (Just asking)

Reply:
Using `nightly` for the `rustfmt` options.